framework/src/base/MooseObject.C

```diff
@@ -20,15 +20,15 @@ template<>
 InputParameters validParams<MooseObject>()
 {
   InputParameters params;
+  params.addParam<std::string>("control_tag", "Adds a user-defined label for accessing object parameters via control logic.");
```

Comment:
I don't think you want people hard-coding tags and objects, that would limit the object to a certain tag. The tags I implemented are object-based, not parameter based.

I need to come up with a different message for tagging parameters with different names. That is not part of this PR, I will add it to my list.